partner_multi_relation/README.rst

```diff
@@ -1,83 +1,166 @@
-one side 'is assistant of' and the other side 'has assistant'. This relation
-only makes sense between people, so you would choose 'Person' for both partner
-types. For the relation 'is a competitor of', both sides would be companies,
-while the relation 'has worked for' should have persons on the left side and
-companies on the right side. If you leave this field empty, the relation is
-applicable to all types of partners.
+Relation Types
+--------------
 
-If you use categories (tags) to further specify the type of partners, you could for
-example enforce that the 'is member of' relation can only have companies with
-label 'Organization' on the left side.
+Before being able to use relations, you'll have define some first.
+Do that in Contacts / Relations / Partner relations.
 
-Now open a partner, click on the 'Relations' smart button and add relations as appropriate.
+.. image:: readme/images/relation_type_list.png
 
-Searching partners with relations
----------------------------------
+A relation type has a name for both sides.
+
+.. image:: readme/images/relation_type_form_empty.png
+
+To have an assistant-relation, you would name one side 'is assistant of' and the other side 'has assistant'.
+
+.. image:: readme/images/relation_type_form_name_filled.png
+
+Partner Types
+-------------
+
+The `Partner Type` fields allow to constrain what type of partners can be used
+on the left and right sides of the relation.
+
+* In the example above, the assistant-relation only makes sense between people, so you would choose 'Person' for both partner types.
+
+.. image:: readme/images/relation_type_form_partner_type_filled.png
+
+* For a relation 'is a competitor of', both sides would be companies.
+* A relation 'has worked for' should have persons on the left side and companies on the right side.
 
-Searching for relations is integrated transparently into the partner search
-form. To find all assistants in your database, fill in 'is assistant of' and
-autocomplete will propose to search for partners having this relation. Now if
-you want to find Anna's assistant, you fill in 'Anna' and one of the proposals
-is to search for partners having a relation with Anna. This results in Anna's
-assistant(s), as you searched for assistants before.
+If you leave these fields empty, the relation is applicable to all types of partners.
 
-By default, only active, not expired relations are shown. If you need to find
-partners that had some relation at a certain date, fill in that date in the
-search box and one of the proposals is to search for relations valid at that
-date.
+Partner Categories
+------------------
 
-More info
+You may use categories (tags) to further specify the type of partners.
+
+You could for example enforce the 'is member of' relation to accept only companies with the label 'Organization' on the right side.
+
+.. image:: readme/images/relation_type_form_category_filled.png
```

Comment:
Shouldn't the images hang from `static`? otherwise the seem to be not reachable?

Reply:
![image](https://user-images.githubusercontent.com/5040182/48564390-92f61600-e8f6-11e8-939e-a41108778a41.png)

Reply:
I'm able to see them:

![seleccion_003](https://user-images.githubusercontent.com/7165771/48565355-a6a27c00-e8f8-11e8-95f0-7f98af438a86.png)

But the problem is that there's no good path for being displayed everywhere: apps store, README, Odoo viewer...

Reply:
My screenshot was taken in the runbot instance app form wich seems to block any static resource linked outside the static folder.

Reply:
It's not only that, but the path: although you put it inside static folder, you have to add the module name prefix, which screws then the visualization of the README on GitHub. I remember an issue about this, but I'm not able to find it. @elicoidal can you refresh my memory?